LORIS_indepth.html

```diff
@@ -290,186 +121,20 @@ <h2>What can LORIS do (pointform)</h2>
-				      </td>
-				      <td width="50%" style="vertical-align: middle">
-					<p><small>
-					    <ul>
-					      <li>MAVAN</li><br>
-					      <li>Maternal Adversity Vulnerability And NeuroDevelopement </li><br>
-					      <li>Behavioral, Imaging and Genomics</li><br>
-					      <li>Key project for the Ludmer Centre</li><br>
-					      <li>Examining acute and chronic stressors on early childhood developement</li><br>
-					     
-					    </ul>
-					</small></p>
-				      </td>
-				  </tr></table>
-				</section>
-			</section>
-
-	<!-- MNI Open Science -->
-		<section>
-
-	 <section data-transition="convex">
-	       <a href="Open_Science.html#/openscience-destination"><h2>Open Science</h2></a>
-	       <table class="reveal">
-		 <tr>
-		   <td width="50%" style="vertical-align: middle">
-			<a href="http://www.sciencemag.org/news/2016/01/montreal-institute-going-open-accelerate-science" target="_blank"><img src="images/Open_science.png"></a>		   
-			</td>
-		   <td width = "50%" style="vertical-align:middle">
-			<img width="85%" src="images/Nature_open_science.png">
-	       </td></tr></table>	
-	     </section>
-
-		<section>
-			<img src="images/TOSI.png">
-		      </section>
-	     
-	     <section>
-	       <h2>Open Science</h2>
-	       <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5216036/" target="_blank"><img src="images/Open_science_paper.png" width="70%"></a>
-	     </section>
-
-<!--Cyberinfrastructure & Cybersecurity -->	     
-	     <section>
-	       <h2>Cyberinfrastructure</h2>
-	       <img src="images/Cyberinfrastructure.png">			
-	     </section>
-
-			<section>
-				<h2>Open Science</h2>
-					<table class="reveal">
-					  <tr>
-					    <td width="50%" style="vertical-align: middle">
-					      	<p><small>
-					    <ul>
-					      <li>Comprises the Biobank and Imaging Centre</li>
-					      <li>LORIS is slated the platform for both</li>
-					      <li>Data sharing at the MNI will be intergral to research</li>
-					      <li>Designed to reduce duplication and make research more efficient</li>
-					    </ul>
-					</small></p>
-					      <p><small><font color="yellow">“We’re doing a really shitty job." -- Guy Rouleau </font></small></p>
-					    </td>
-					    <td>
-					      <img src="images/Open_science.png"> 
-					</td></tr></table>	
-			</section>
-        </section>
-
-		    <!-- LORIS core -->
+            
+            <section data-background="images/Rogers_poster.PNG" data-background-size="70%">
+                    </section>
+            
+              </section>
+              
+<!-- LORIS high-level overview Column 3 -->
 		    <section>
+                
 		      <section>
-                        <h2>Login</h2>
-                        <a href="https://demo.loris.ca" target="_blank"><img src="images/LORIS_login.png"></a>
+                        <p>High-Level Walkthrough of LORIS</p>
```

Comment:
needs to be a header font

Reply:
done - I tried to avoid it sometimes since it defaults all letters to caps. not sure how to change that!